UCSLibraries/src/main/java/it/cnr/iit/ucsinterface/contexthandler/ContextHandlerPIPInterface.java

```diff
@@ -15,24 +15,24 @@
  ******************************************************************************/
 package it.cnr.iit.ucsinterface.contexthandler;
 
-import it.cnr.iit.ucsinterface.message.Message;
+import it.cnr.iit.ucsinterface.message.pipch.PipChMessage;
```

Comment:
May be we should rename this class to PIPCHMessage as everywhere else we use capitals as PIP or CH etc

Reply:
yes could we fine a better name?

Reply:
Something like AttributeChangedMessage ?